### PR TITLE
Magiclysm : Lost Faith scenario

### PR DESCRIPTION
**Pull request by Leosky**
#### Summary
```SUMMARY: Mods "Magiclysm - Lost Faith scenario"```

#### Purpose of change
Add a basic scenario with spellbook in a surrounded city start location .

#### Describe the solution
-

#### Describe alternatives you've considered
-

#### Additional context
The 'Church' location is the only temple currently provided and the scenario may be altered to add other religion cult place.

### CODE REVIEW

**Comment by KorGgenT** [comment]
You need to remove the brackets around Mods in the summary

File: data/mods/Magiclysm/scenarios.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "ident": "lost_faith",
+    "name": "Lost Faith",
+    "flags": [ "SUR_START", "LONE_START" ],
+    "points": 2,
+    "description": "You've been preaching your faith during a long time, but the recent events make you doubt about it. Wandering in your temple, you just found books that may restore your faith or finish to shatter it.",
+    "start_name": "Church",
+    "allowed_locs": [ "church_3rdfloor_1", "church_1", "church_steeple_end", "church" ],
+    "professions": [ "priest_magic2", "shinto_priest_magic2", "imam_magic2", "rabbi_magic2", "guru_magic2", "preacher_magic2" ]
+  }
+]
```
Comment (ZhilkinSerg) [inline data/mods/Magiclysm/scenarios.json]:
```suggestion
]

```

**Comment by KorGgenT** [comment]
looks like you need to lint your files.

File: data/mods/Magiclysm/scenarios.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "type": "scenario",
+    "ident": "lost_faith",
+    "name": "Lost Faith",
+    "flags": [ "SUR_START", "LONE_START" ],
+    "points": 2,
+    "description": "You've been preaching your faith during a long time, but the recent events make you doubt about it. Wandering in your temple, you just found books that may restore your faith or finish to shatter it.",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/scenarios.json]: I don't like the wording here. "finish to shatter it" is very clunky.

File: data/mods/Magiclysm/professions.json [hunk 3]
```diff
@@ -34,5 +34,145 @@
       "both": [ "druid_spellbook", "leathersandals", "robe", "rope_makeshift_30", "hide_bag", "hat_fur", "gloves_wraps_fur" ]
     },
     "traits": [ "DRUID" ]
+  },
+  {
+    "type": "profession",
+    "ident": "priest_magic2",
+    "name": "Priest",
+    "description": "When the apocalypse struck, you did everything you could to protect your parish faithful, but it appears that prayers were not enough.  Now that they are all dead, you should probably find something more tangible to protect you.",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/professions.json]:
```suggestion
    "description": "When the apocalypse struck you did everything you could to protect your parish faithful, but it appears that prayers were not enough.  Now that they are all dead, you should probably find something more tangible to protect you.",
```

File: data/mods/Magiclysm/professions.json [hunk 4]
```diff
@@ -34,5 +34,145 @@
...
+      },
+      "male": [ "briefs" ],
+      "female": [ "bra", "panties" ]
+    }
+  },
+  {
+    "type": "profession",
+    "ident": "imam_magic2",
+    "name": { "male": "Imam", "female": "Mourchida" },
+    "description": "You spent much of your time prior to the apocalypse at the local mosque, studying the words of the Prophet and the Quran, and guiding your community in prayer.  Back then they came from far and wide to listen to you, now they come to eat your brains.",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/professions.json]:
```suggestion
    "description": "You spent much of your time prior to the apocalypse at the local mosque, studying the words of the Prophet and the Quran and guiding your community in prayer.  Back then they came from far and wide to listen to you, now they come to eat your brains.",
```

File: data/mods/Magiclysm/scenarios.json (hunk 2 above)
Comment (Leosky) [reply data/mods/Magiclysm/scenarios.json]: "erode it even more" ?

File: data/mods/Magiclysm/professions.json [hunk 5]
```diff
@@ -34,5 +34,145 @@
...
+      },
+      "male": [ "briefs" ],
+      "female": [ "bra", "panties" ]
+    }
+  },
+  {
+    "type": "profession",
+    "ident": "guru_magic2",
+    "name": "Guru",
+    "description": "You spent many years traveling through the world, becoming wise and learned.  Normally, you can answer any question, but even you are not quite sure what to do about the ravenous undead.",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/professions.json]:
```suggestion
    "description": "You spent many years traveling the world, becoming wise and learned.  Normally, you can answer any question, but even you are not quite sure what to do about the ravenous undead.",
```

File: data/mods/Magiclysm/professions.json [hunk 6]
```diff
@@ -34,5 +34,145 @@
...
+      },
+      "male": [ "briefs" ],
+      "female": [ "panties" ]
+    }
+  },
+  {
+    "type": "profession",
+    "ident": "preacher_magic2",
+    "name": "Preacher",
+    "description": "You devoted your life to spreading the good word, always on the road, traveling from town to town.  Now, everything has gone to hell, you can't host your daily podcast and the undead listening to your sermons don't seem particularly moved.",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/professions.json]:
```suggestion
    "description": "You devoted your life to spreading the good word; always on the road, traveling from town to town.  Now everything has gone to hell, you can't host your daily podcast, and the undead listening to your sermons don't seem particularly moved.",
```

File: data/mods/Magiclysm/scenarios.json (hunk 2 above)
Comment (KorGgenT) [reply data/mods/Magiclysm/scenarios.json]: "shatter it completely"?

File: data/mods/Magiclysm/scenarios.json [hunk 7]
```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "type": "scenario",
+    "ident": "lost_faith",
+    "name": "Lost Faith",
+    "flags": [ "SUR_START", "LONE_START" ],
+    "points": 2,
+    "description": "You've been preaching your faith during a long time, but the recent events make you doubt about it. Wandering in your temple, you just found books that may restore your faith or to shatter it completely.",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/scenarios.json]:
```suggestion
    "description": "You've been preaching your faith during a long time, but the recent events make you doubt about it. Wandering in your temple, you just found books that may restore your faith or shatter it completely.",
```

**Pull request merged**